internal/resolver/dns/dns_resolver.go

```diff
@@ -45,6 +45,13 @@ import (
 // addresses from SRV records.  Must not be changed after init time.
 var EnableSRVLookups = false
 
+// ResolvingTimeout specifies the maximum duration for a DNS resolution request.
+// If the timeout expires before a response is received, the request will be canceled.
+//
+// It is recommended to set this value at application startup. Avoid modifying this variable
+// after initialization as it's not thread-safe for concurrent modification.
```

Comment:
```suggestion
// ResolvingTimeout specifies the maximum duration for a DNS resolution request.
// If the timeout expires before a response is received, the request will be
// canceled.
//
// It is recommended to set this value at application startup. Avoid modifying
// this variable after initialization as it's not thread-safe for concurrent
// modification.
```

To keep the comment line length consistent to 80-column wide